dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/task/DumpMetadataTask.java

```diff
@@ -34,18 +36,29 @@ public class DumpMetadataTask extends AbstractTask<Void>
   @Nullable private final ConnectorArguments arguments;
   private final String format;
 
+  @Nullable private final ImmutableList<String> connectors;
```

Comment:
Could this list be non-null (using empty list where null is currently used)?

Reply:
No, with the empty list the Yaml output is `connectors: []` instead of not including the property. This property makes sense only in the context of metaconnector and I don't want to change the format in other connectors.